tools/rebuild-storage-service-db

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function
+
+import argparse
+import json
+import os
+import re
+import requests
+import sys
+
+
+def main(path, ss_url, space, location, pipeline):
+    api_version = '2'
+
+    # Check if Space exists
+    url = 'http://{}/api/v{}/space/{}/'.format(ss_url, api_version, space)
+    response = requests.get(url, verify=False)
+    if response.status_code == 404:
+        # Create new local space
+        print('TODO create local space')
+        return 2
+    elif response.status_code == 200:
+        print('Space', space, 'exists')
+        space_info = response.json()
+
+    # Check if Location exist
+    url = 'http://{}/api/v{}/location/{}/'.format(ss_url, api_version, location)
+    response = requests.get(url, verify=False)
+    if response.status_code == 404:
+        # Create new location
+        print('TODO create location within', space)
+        return 2
+    elif response.status_code == 200:
+        # Check if Location is AS
+        location_info = response.json()
+        print('Location', location, 'exists')
+        if location_info.get('purpose') != 'AS':
+            print(location, 'is not an AIP storage location. Exiting', file=sys.stderr)
+            return 1
+        location_info = response.json()
+
+    # Check if Pipeline exists
+    url = 'http://{}/api/v{}/pipeline/{}/'.format(ss_url, api_version, pipeline)
+    response = requests.get(url, verify=False)
+    if response.status_code == 404:
+        # Create new local pipeline
+        print('TODO create local pipeline')
+        return 2
+    elif response.status_code == 200:
+        print('Pipeline', pipeline, 'exists')
+        pipeline_info = response.json()
+
+    # Walk directory
+    print('Searching', path, 'for AIPS...')
+    aip_name_regex = r'^(?P<name>\w*)-(?P<uuid>\w{8}-\w{4}-4\w{3}-\w{4}-\w{12}).(?P<extension>7z|tar.bz2)$'
```

Comment:
Will `\w` match all characters that can be in an AIP name? I think `-` is also valid.